uaclient/cli.py

```diff
@@ -377,6 +387,10 @@ def refresh_parser(parser):
     return parser
 
 
+def action_update_info(args, *, cfg, **kwargs):
+    return json.dumps(update_info.update_info(cfg))
```

Comment:
We are only outputting this in json in the first iteration, right ? We want this to be user readable as well at some point in time, right ?